Server/src/services/DataSetService.ts

```diff
@@ -544,7 +544,7 @@ export class DataSetService {
                 throw new InternalServerError("Internal server error approving this data set. Try again later")
             }
             else {
-                throw new Error()
+                throw new Error(error.message)

```

Comment:
generic error?

Reply:
Yeah. For any other possible thrown error